src/System.Private.CoreLib/src/System/Reflection/Emit/MethodBuilder.cs

```diff
@@ -87,12 +87,12 @@ public sealed class MethodBuilder : MethodInfo
             m_module = mod;
             m_containingType = type;
 
-            // 
-            //if (returnType == null)
-            //{
-            //    m_returnType = typeof(void);
-            //}
-            //else
+
+            if (returnType == null)
+            {
+                m_returnType = typeof(void);
+            }
+            else
             {
                 m_returnType = returnType;
             }
```

Comment:
Nit: could be simplified to `m_returnType = returnType ?? typeof(void);`

Reply:
Fixed.